### DISCUSSION TITLE
发送的转发消息变成了json的了，该怎么获取里面的内容

### DISCUSSION THREAD

**Discussion by XiaoFuOS** [Mrs4s/go-cqhttp#2175] [Q&A]
我记得前段时间发的转发消息都还是CQ:forward的，现在怎么是json得了，要怎么解析里面的内容鸭

![image](https://github.com/Mrs4s/go-cqhttp/assets/103229747/f4823bf2-0d28-4bbb-8757-1dff7946e836)

**Comment by yyx2684264** [comment]
这里面有个resid字段，解析出来后按照原本的获取转发消息方法获取就行了
